docs/source/user_manual/basic_elements/tools.rst

```diff
@@ -126,8 +126,8 @@ The event data contains 3 keys: `indices` with the 2D coordinates of the mouse
 in data space, `color_value` containing the color of the tile where the mouse
 is, and `data_value` with the scalar value being displayed in that tile.
 
-For a complete example, see :download:`examples/demo/basic/image_inspector.py
-<../../../examples/demo/basic/image_inspector.py>`.
+For a complete example, see :download:`chaco/examples/demo/basic/image_inspector.py
+<../../../chaco/examples/demo/basic/image_inspector.py>`.
```

Comment:
i should have built the docs and tested before approving this PR but the link in the tools page is still broken on #792 .